Frontend/src/CustomScripts/convertData.mjs

```diff
@@ -88,7 +91,50 @@ const json = { categories, defaultHistoryLength };
 
 // write this JSON object
 const output = JSON.stringify(json, null, 2);
```

Comment:
```suggestion
```
It looks like `output` isn't used, so it can be removed.

Reply:
fixed